staging-infra.yaml

```diff
@@ -0,0 +1,28 @@
+args:
+  configurationKey:
```

Comment:
Could we somehow leave these inputs out? They are not relevant for this deployment.

Reply:
That seems to require some meta-information to be attached to the properties -- which can then be used to filter the generated config.

Do you think that would be worth the complication?